src/otx/core/model/visual_prompting.py

```diff
@@ -596,7 +584,7 @@ def _set_label_info(self, _: LabelInfoTypes) -> None:
     def get_dummy_input(self, batch_size: int = 1) -> ZeroShotVisualPromptingBatchDataEntity:
         """Returns a dummy input for ZSL VPT model."""
         images = [torch.rand(3, *self.input_size) for _ in range(batch_size)]
-        labels = [ZeroShotVisualPromptingLabel(prompts=torch.LongTensor([0]))] * batch_size
+        labels = [{"points": torch.LongTensor([0] * batch_size)}] * batch_size
```

Comment:
```suggestion
        labels = [{"points": torch.LongTensor([0] * batch_size)}] * batch_size
```
I'm not sure but is it ok to multiply batch_size twice?

Reply:
It doesn't matter to use it twice because it is just to create dummy inputs for exportation.